### DISCUSSION TITLE
Using CommunityToolkit.Maui with CommunityTookit.Mvvm update errors

### DISCUSSION THREAD

**Discussion by devonuto** [CommunityToolkit/Maui#443] [Q&A] [answered]
I have been using both together for a while (not sure if this is expected but it was working).

"CommunityToolkit.Mvvm" Version="8.0.0-preview3 was all good, but after an update came through for preview4, I now get all these errors:
Error	CS0246	The type or namespace name 'AlsoNotifyChangeFor' could not be found 
Error	CS0246	The type or namespace name 'AlsoNotifyChangeForAttribute' could not be found 
Error	CS0246	The type or namespace name 'ICommand' could not be found 
Error	CS0246	The type or namespace name 'ICommandAttribute' could not be found 

Have these been removed? Are they renamed?
Am I not supposed to be using Mvvm with Maui?

**Comment by TheCodeTraveler** [comment, marked as answer]
Yup! The APIs changed in the MVVM Toolkit v8.0.0-pre4:
- `[AlsoNotifyChangeFor]` -> `[NotifyPropertyChangedFor]`
- `[ICommand]` -> `[RelayCommand]`

As an example, here's the pull request I merged today for my own app, GitTrends, updating it to MVVM Toolkit v8.0.0-pre4: https://github.com/brminnick/GitTrends/pull/477

**Reply by devonuto** [reply]
Excellent. Working again, thanks!

**Comment by CliffAgius** [comment]
Closed as answered
